src/main/java/org/nrnb/gsoc/enrichment/tasks/EnrichmentTask.java

```diff
@@ -393,9 +393,10 @@ else if (ModelUtils.getNetSignificanceThresholdMethod(network)!=null && signific
 	}
 
 
-	public void cancel() {
-		this.cancelled = true;
-	}
+    public void cancel() {
+        ScheduledRequestEngine.stopPostRequest();
+        this.cancelled = true;
```

Comment:
Rename this variable to something that indicates what it is being used for(Example:`this.gProfilerCallCancelled`)  and declare it in the class along with the other variables which have been declared(`private boolean gProfilerCallCancelled`).

Reply:
`cancelled` variable is inherited from `AbstractTask`, so I am afraid we cannot rename it.